### PR TITLE
Shooter, fixing teleop, etc

### PR DESCRIPTION
**Pull request by JacksonMcLaughlin**
shooter subsystem and shoot command c/o @EmmiWyttenbach. also fixing a lot of my mistakes

### CODE REVIEW

File: src/org/usfirst/frc/team7171/robot/OI.java [hunk 1]
```diff
@@ -40,11 +42,15 @@
 	public static Joystick jsTurn = new Joystick(RobotMap.JS_TURN);
 	
 	public static Joystick coDriverButtons = new Joystick(RobotMap.CD_BUTTONS);
-	private static final int N_DEADMAN = 0;
+	public static Joystick DriverButtons = new Joystick(RobotMap.D_BUTTONS);
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/OI.java]: camel case best case? `driverButtons`

File: src/org/usfirst/frc/team7171/robot/OI.java [hunk 2]
```diff
@@ -40,11 +42,15 @@
...
+	private static final int N_DEADMAN = 1;
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/OI.java]: what does the `N_` prefix mean?

File: src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import org.usfirst.frc.team7171.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ShootCommand extends Command {
+
+	private boolean extend;
+	
+	public ShootCommand(boolean e)
+	{
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java]: mixing the two brace styles is punishable by robot law

File: src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		extend = e;
+	}
+	@Override
+	protected void initialize() {
+		Robot.s.setExtended(extend);
+		
+	}
+	@Override
+	protected boolean isFinished() {
+		return isTimedOut();
```
Comment (octylFractal) [inline src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java:24]: doesn't this happen in `Command` already?

**Review by octylFractal — CHANGES_REQUESTED** [review]

File: src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java (hunk 4 above)
Comment (JacksonMcLaughlin) [reply src/org/usfirst/frc/team7171/robot/commands/ShootCommand.java:24]: you still have to implement isFinished and return something

**Pull request merged**